app/models/course.rb

```diff
@@ -156,6 +156,10 @@ def self.unsubmitted
 
   scope :ready_for_update, -> { current.or(where(needs_update: true)) }
 
+  scope :ready_for_short_update, lambda {
+    where(type: 'Editathon').where(needs_update: true)
```

Comment:
The `needs_update` attribute is only ever set manually for an individual course (by clicking the 'update stats' button that appears after the event has ended), and it gets reset to false after the next time it gets updated.

In this case, I think we could something like `strictly_current.where(type: 'Editathon')`. That would only include editathon events while they are going on. Alternatively, instead of basing it on type, you could write a scope based on the program being short or nearly over: strictly_current ending in less than a day from now, for example.